pkg/controller/scylladbcluster/resource.go

```diff
@@ -434,6 +434,44 @@ func calculateEndpointsForRemoteDCPods(sc *scyllav1alpha1.ScyllaDBCluster, nodeB
 			})
 		}
 
+	case scyllav1alpha1.BroadcastAddressTypeServiceClusterIP:
+		dcServices, err := remoteServiceLister.Cluster(remoteDC.RemoteKubernetesClusterName).Services(remoteDCNamespace.Name).List(remoteDCPodSelector)
+		if err != nil {
+			return nil, fmt.Errorf("can't list services in %q ScyllaDBCluster %q Datacenter: %w", naming.ObjRef(sc), remoteDC.Name, err)
+		}
+
+		klog.V(4).InfoS("Found remote ScyllaDB Services", "ScyllaDBCluster", klog.KObj(sc), "Datacenter", remoteDC.Name, "Services", len(dcServices))
+
+		// Sort objects to have stable list of endpoints
+		sort.Slice(dcServices, func(i, j int) bool {
+			return dcServices[i].Name < dcServices[j].Name
+		})
+
```

Comment:
It's copy pasted from LB case, lets try to deduplicate it by extracting function iterating over each service and extracting necessary information or returning an endpoint.